src/events/Dürrewelle.ts

```diff
@@ -1,16 +1,20 @@
 import { defineEvent } from "../Factory"
 import { markdown } from "../lib/utils"
+import { modify } from "../params"
 
 export default defineEvent({
-  title: "",
-  description: ``,
+  title: "Dürreperiode",
+  description: `Die anhaltende Trockenheit und die damit verbundene Wasserknappheit hatte Rationierungen zur Folge. Die Ernteausfälle sind enorm.`,
 
   apply() {
-    return []
+    return [
+      modify("gdp").byValue(-100),
+      modify("popularity").byValue(-10),
+    ]
   },
 
-  probability() {
-    return Math.random()
+  probability(game) {
+    return Math.min(1, Math.max(0, (400 - game.values.co2budget) / 400))
```

Comment:
You could use the `linear()` function in lawTools for this - or rather a modified version for events - sad thad we have between 0 and 1 here and between 0 and 100 in the laws...

Reply:
...but it is actually a pretty nice idea to trigger such events by an almost used-up co2budget.

Reply:
We can later change events to us 0..100 as well. Also, we can change the name of the `lawTools` library to something else, to make it more clear, that it isn't restricted to laws. Or move the function somewhere else.
For now, I will only use `linear()`.